stack.yaml

```diff
@@ -1,17 +1,117 @@
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/cb54482c7e3d45f6c1aac490dc937ed835de0333/snapshot.yaml
+resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/00487726c4bc21b4744e59d913334ebfeac7d68e/snapshot.yaml
```

Comment:
this absolutely looks wrong

Reply:
Reverted to the resolver from master with commit 860e263